shared/data/constants.js

```diff
@@ -5,10 +5,10 @@ function getConfigFileName () {
     let browserName = browserInfo?.browser?.toLowerCase() || ''
 
     // clamp to known browsers
-    if (!['chrome', 'firefox', 'brave', 'edge'].includes(browserName)) {
+    if (!['chrome', 'firefox', 'brave', 'edg'].includes(browserName)) {
```

Comment:
Was edge the wrong name or did it change?

Reply:
When edge switched to chromium they changed to 'Edg' to differentiate with the original Edge engine. 'Edge' shouldn't exist in the wild now that that browser is deprecated.

Reply:
We should rename the regular edg file also then: https://github.com/duckduckgo/privacy-configuration/commit/ebe3b9cf74555abf67ab09a51399782ef38d92e7

Reply:
Opened a PR for that: https://github.com/duckduckgo/privacy-configuration/pull/531